Cookbook/Technical-Documents/NewHiresCheckList.md

```diff
@@ -42,52 +42,23 @@ As an iOS Engineer, you should be in the following Slack channels:
 
 Make sure you also join your Tribe/Squad's Slack channels.
 
-## Setup your development environment
+## Install prerequisites.
 
-Here's how to get the iOS project up and running.
+  - Xcode + its command-line tools
+  - [Homebrew](https://brew.sh/)
+  - Ruby 2.4 (you can use [RVM](https://rvm.io/) to install it and manage your Ruby versions)
+
+## Set up your development environment.
+
+Most of the work to get the iOS project up and running is automated inside a
+shell script. Here's how to get the iOS project up and running:
 
-1. Globally configure Git to use SSH instead of HTTPS:
-     ```
-     git config --global url."git@github.com:".insteadOf "https://github.com/"
-     ```
-     
 1. [Generate](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#generating-a-new-ssh-key) & add the SSH key to your [GitHub account](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account)
 
-1. Install Homebrew if you don't already have it installed:    
-     ```
-     /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-     ```
-
-1. Setup Git LFS:
-     ```
-     brew install git-lfs
-     git lfs install
-     ```
-
-1. Clone the iOS repository: https://github.com/Babylonpartners/babylon-ios  
-*Make sure you've already done the `Setup Git LFS` step before cloning the project.*
-     ```
-     git clone git@github.com:Babylonpartners/babylon-ios.git
-     ```
-
-1. Get our latest Xcode templates by running `./Templates/install_xcode_templates.sh`
-
-1. Install RVM and Ruby version 2.4
```

Comment:
We can preserve RVM + Ruby 2.4 install manual for better tutorial, though RVM is not always necessary.

Reply:
I agree with @inamiy. IMO we should keep the Ruby + RVM installation instructions here because it's not immediately obvious from the RVM site what you should do.

Also, https://rvm.io is currently down for me, but the `get.rvm.io` script is not.

Reply:
Good point. I added back the instructions.